lib/src/cli/flutter_cli.dart

```diff
@@ -165,14 +167,15 @@ class Flutter {
         void noop(String? _) {}
         final target = DirectoryGeneratorTarget(Directory(p.normalize(cwd)));
         final workingDirectory = target.dir.absolute.path;
+        final relativePath = p.relative(workingDirectory, from: initialCwd);
 
         stdout?.call(
-          'Running "flutter test" in ${p.dirname(workingDirectory)}...\n',
+          'Running "flutter test" in .${p.context.separator}$relativePath...\n',
```

Comment:
We should probably add a space between `$relativePath` and the `...` used as progress indicators to avoid confusing the user with four consecutive fullstops `....` when `$relativePath` is "." (as in some of the tests).

```suggestion
          'Running "flutter test" in .${p.context.separator}$relativePath ...\n',
```